### PR TITLE
Latest modules for master

### PR DESCRIPTION
**Pull request by chayim**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 
 # module versions
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
-  redisearch: 2.6.1
```
Comment (ofekshenawa) [inline config.yml:7]: search looks like 2.6.2
https://github.com/RediSearch/RediSearch/releases

File: config.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
 
 # module versions
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
```
Comment (ofekshenawa) [inline config.yml:6]: Timeseries looks like 2.0.0
https://github.com/RedisTimeSeries/RedisTimeSeries/releases

File: config.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-  redisearch: 2.6.1
-  redisgraph: 2.10.2
-  redisbloom: 2.4.2
```
Comment (ofekshenawa) [inline config.yml:9]: 3.0.0
https://github.com/RedisBloom/RedisBloom/releases

File: config.yml [hunk 4]
```diff
@@ -1,12 +1,12 @@
 ---
 
 # module versions
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
-  redisearch: 2.6.1
-  redisgraph: 2.10.2
```
Comment (ofekshenawa) [inline config.yml:8]: 3.0.0 - graph
https://github.com/RedisGraph/RedisGraph/releases

File: config.yml [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
-  redisbloom: 2.4.2
+versions1
+  rejson: 2.4.1
+  redistimeseries: 1.8.3
+  redisearch: 2.6.2
+  redisgraph: 2.10.3
```
Comment (ofekshenawa) [inline config.yml:8]: 3.0.0 - graph
https://github.com/RedisGraph/RedisGraph/releases

File: config.yml [hunk 6]
```diff
@@ -1,12 +1,12 @@
...
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
-  redisearch: 2.6.1
-  redisgraph: 2.10.2
-  redisbloom: 2.4.2
+versions1
+  rejson: 2.4.1
+  redistimeseries: 1.8.3
+  redisearch: 2.6.2
```
Comment (ofekshenawa) [inline config.yml:7]: search looks like 2.6.2
https://github.com/RediSearch/RediSearch/releases

File: config.yml [hunk 7]
```diff
@@ -1,12 +1,12 @@
...
 # module versions
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
-  redisearch: 2.6.1
-  redisgraph: 2.10.2
-  redisbloom: 2.4.2
+versions1
+  rejson: 2.4.1
+  redistimeseries: 1.8.3
```
Comment (ofekshenawa) [inline config.yml:6]: Timeseries looks like 2.0.0
https://github.com/RedisTimeSeries/RedisTimeSeries/releases

File: config.yml [hunk 8]
```diff
@@ -1,12 +1,12 @@
...
 
 # module versions
-versions:
-  rejson: 2.4.0
-  redistimeseries: 1.8.2
-  redisearch: 2.6.1
-  redisgraph: 2.10.2
-  redisbloom: 2.4.2
+versions1
+  rejson: 2.4.1
```
Comment (ofekshenawa) [inline config.yml:5]: json looks like 2.4.1
https://github.com/Redisjson/Redisjson/releases

File: config.yml [hunk 9]
```diff
@@ -1,12 +1,12 @@
...
+  redistimeseries: 1.8.3
+  redisearch: 2.6.2
+  redisgraph: 2.10.3
+  redisbloom: 2.4.3
```
[the remark this answers is not in the document]
Comment (chayim) [reply config.yml:9]: No, they decided on 2.4.3

File: config.yml (hunk 5 above)
Comment (chayim) [reply config.yml:8]: No, they decided on 2.10.3 everywhere... these are other draft releases

**Review by ofekshenawa — APPROVED** [review]

**Pull request merged**